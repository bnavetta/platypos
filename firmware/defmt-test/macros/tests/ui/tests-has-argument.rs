fn main() {}

#[defmt_test_macros::tests(tests)]
mod tests {
}
