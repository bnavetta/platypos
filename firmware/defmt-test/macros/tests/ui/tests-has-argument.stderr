error: `#[test]` attribute takes no arguments
 --> tests/ui/tests-has-argument.rs:3:1
  |
3 | #[defmt_test_macros::tests(tests)]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `defmt_test_macros::tests` (in Nightly builds, run with -Z macro-backtrace for more info)
